microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -169,8 +169,7 @@ You can delete templates stored on the device and clear the cache data. Go to `e
 
 <!-- Above, should pick one way. Check UI for privacy updates -->
 
-You can use the title of your collection to find relevant Pinterest Topic pages when searching in Microsoft Bing. Go to `edge://settings/privacy`. Turn on the **Show suggestions from Pinterest in Collections** setting. Microsoft Edge does not send data about your collections to Pinterest. You can remove the suggestions and stop searches for Pinterest Topic pages. Go to `edge://settings/privacy` and turn off the **Show suggestions from Pinterest in Collections** setting.
-Collections aren't available when using **InPrivate** or **Guest** modes.
+If you're signed in with a **personal Microsoft account**, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, go to `edge://settings/privacy`, <!--todo: scroll down to which section?--> and then turn off the **Show Collections and follow content creators in Microsoft Edge**<!--todo: no such setting--> toggle.  When this setting is turned off, Collections are disabled on this device.  Collections aren't available when using Guest mode or InPrivate browsing for your **personal Microsoft account**.
```

Comment:
Regarding: "automatically saved on Microsoft cloud for free". I found the following statement in another part of the Collections section of the whitepaper: "All collections data is stored locally on the device and organized per Microsoft Edge profile. If you have sync turned on for Collections, your collections, notes, and comments are available across all signed-in and syncing versions of Microsoft Edge.". Are these two statements in opposition, or are they both true? Either way, I think this addition to the whitepaper requires us to edit that section as well. With your clarification, I can help edit/re-write.

Reply:
@yuanjunz - see above comment

Reply:
To help you understand, below are background
We have 3 types of Collections data.

- 1, For non sign in users, they are using Classic collections without sync. So statement: All collections data is stored locally on the device and organized per Microsoft Edge profile.

- 2, For AAD sign in users, they are using Classic collections (legacy mode). 
- - if sync is not turned on. Their collections data is stored locally. 
- - if sync is turned on. Local stored collections data will sync to Edge sync cloud. 

- 3 For MSA sign in users, 
- - if sync is not turned on. Their collections data is stored locally. 
- - if sync is turned on. Their Collections data is updated to New Collections (migrated to Prism collections). And synced to Microsoft Bing Saves cloud.

So, "All collections data is stored locally on the device and organized per Microsoft Edge profile." => means all Classic Collections (legacy Collections) data are stored locally, except new Collections (Prism collections). Above # 1

"If you have sync turned on for Collections, your collections, notes, and comments are available across all signed-in and syncing versions of Microsoft Edge." => means for AAD signed in and sync turned on users, Collection data is synced from local store to Edge sync cloud. Above # 2.

"If you're signed in with a Microsoft account, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free" => Means for MSA signed in and sync turned on users. new Collections (Prism Collections) data is saved to Microsoft Bing Saves cloud. Above # 3

Reply:
@apryzgoda - I added the above info, and added hidden comments to identify Sentence A, B, & C, displayed as quoted sentences near bottom of section (like in the above comment).  We want to not quote those entire sentences A, B, & C.  Ideas for reorganizing the Collections section?

Reply:
Incorp'd rewrite from Alyssa